src/Aquifer.API/Endpoints/Projects/ProjectResourceStatusCounts.cs

```diff
@@ -39,9 +39,30 @@ public class ProjectResourceStatusCounts
         ResourceContentStatus.Complete, ResourceContentStatus.CompleteNotApplicable
     ];
 
+    internal static readonly List<ResourceContentStatus> RemainingStatuses =
+    [
+        ResourceContentStatus.AquiferizeAiDraftComplete,
+        ResourceContentStatus.AquiferizeEditorReview,
+        ResourceContentStatus.TranslationEditorReview,
+        ResourceContentStatus.AquiferizeCompanyReview,
+        ResourceContentStatus.TranslationCompanyReview
+    ];
+
+    public ProjectResourceStatusCounts() { }
+
+    public ProjectResourceStatusCounts(List<(ResourceContentStatus Status, int? WordCount)> resourceData)
```

Comment:
Using an `IReadOnlyList<T>` allows callers to pass an `Array<T>`, a `List<T>`, or other implementing types, which makes it easier for caller to use this method.  I generally choose `IEnumerable<T>` or `IReadOnlyList<T>` depending upon if my method needs to enumerate the items more than once.
```suggestion
    public ProjectResourceStatusCounts(IReadOnlyList<(ResourceContentStatus Status, int? WordCount)> resourceData)
```